src/OrchardCore.Modules/OrchardCore.Contents/Controllers/AdminController.cs

```diff
@@ -102,16 +102,15 @@ public async Task<IActionResult> List(ListContentsViewModel model, PagerParamete
                 }
 
                 // Allows non creatable types to be created by another admin page.
-                if (model.Options.CanCreateSelectedContentType)
+                if (contentTypeDefinition.GetSettings<ContentTypeSettings>().Creatable || model.Options.CanCreateSelectedContentType)
                 {
                     model.Options.CreatableTypes = new List<SelectListItem>
                     {
                         new SelectListItem(new LocalizedString(contentTypeDefinition.DisplayName, contentTypeDefinition.DisplayName).Value, contentTypeDefinition.Name)
                     };
                 }
```

Comment:
So, if there is a `SelectedContentType`, because of the inner condition the `CreatableTypes` may be still null, and because of the `else` line 113 the `CreatableTypes` may not be initialized and it will fail in `ContentsAdminList-Create.cshtml`.

We could do a null check in the view but i suggest here to add an `else` to init `CreatableTypes` to an empty list.

Reply:
I have changed the code.
I hope to have understand your suggestion. Thank you @jtkech.